test/fetch.js

```diff
@@ -317,6 +317,36 @@ t.test('removes authorization header if changing hostnames', async (t) => {
   t.notOk(httpsSrv.isDone(), 'redirect request does not happen')
 })
 
+t.test('removes cookie header if changing hostnames', async (t) => {
+  const httpSrv = nock(HTTPHOST, {
+    reqheaders: {
+      cookie: 'test=true',
+    },
+  })
+    .get('/redirect')
+    .reply(301, '', { Location: `${HOST}/test` })
+
+  const httpsSrv = nock(HOST, {
+    reqheaders: {
+      cookie: 'test=true',
+    },
+  })
+    .get('/test')
+    .reply(200, () => {
+      t.equal(true, false, 'meaningful failure, this should never be executed')
```

Comment:
can also use `t.fail()` here instead 😊

Reply:
I copied the other header test, this was not the PR in which to be iterating on the tests